charts/ccd-definition-store-api/values.preview.template.yaml

```diff
@@ -1,5 +1,21 @@
 java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  aadIdentityName:
+  keyVaults:
+    ccd:
+      secrets:
+        - ccd-definition-s2s-secret
```

Comment:
Might need to do terraform change first to get this PR working ?

Reply:
I've copied manually into ccd AAT vault to get this working. Once this PR is merged terraform will override that value.  Basically just combined these changes together in this PR as waiting for another merge in ccd is painful 😉